src/test/java/com/hocs/test/pages/decs/Search.java

```diff
@@ -632,6 +632,7 @@ public void assertCOMPInformationRandomSearchResult(String criteria) {
         WebElementFacade randomSearchResultHypertext = findBy("//tr[" + randomNumber + "]/td/a");
         String randomSearchResult = randomSearchResultHypertext.getText();
         setSessionVariable("randomCaseRef").to(randomSearchResult);
+        waitFor(randomSearchResultHypertext);
```

Comment:
this has no purpose. If the element isn't present then line 633 would fail before we got to this